ui/app/src/components/Breadcrumb.tsx

```diff
@@ -14,41 +14,53 @@
 import { Breadcrumbs, Link, Typography } from '@mui/material';
 import { Link as RouterLink } from 'react-router-dom';
 
-interface DashboardBreadcrumbsProps {
-  dashboardProject?: string;
+interface BreadcrumbProps {
+  projectName?: string;
   dashboardName?: string;
+  admin?: boolean;
 }
 
 /*
- * DashboardBreadcrumbs provide a naviguation helper
- * For dashboard breadcrumb, dashboardProject & dashboardName are mandatory
- * For project breadcrumb, dashboardProject is mandatory
+ * Breadcrumb provide a naviguation helper
+ * For dashboard breadcrumb, projectName & dashboardName are mandatory
+ * For project breadcrumb, projectName is mandatory
  * For home breadcrumb, all props need to be empty or undefined
  */
-function DashboardBreadcrumbs(props: DashboardBreadcrumbsProps) {
-  const { dashboardProject, dashboardName } = props;
+function Breadcrumb(props: BreadcrumbProps) {
```

Comment:
It looks like Breadcrumbs take an s at the end. I didn´t know that but I would trust material on this.